features/build/buildconfig.feature

```diff
@@ -79,30 +79,30 @@ Feature: buildconfig.feature
     And the "ruby-sample-build-1" build was created
     Then the "ruby-sample-build-1" build completes
 
-    Examples:
-      | template                              |
-      | tc479541/test-buildconfig-s2i.json    | # @case_id OCP-11474
+    Examples: 
+      | template                           |
+      | tc479541/test-buildconfig-s2i.json |
```

Comment:
Definitely, this is not something we want.

Reply:
Workarounds,
-  We skip those feature files which contains `Scenario Outline`.
- We change the format to
```
    Examples: 
      | template                           |
      # @case_id OCP-11474
      | tc479541/test-buildconfig-s2i.json |
      # @case_id OCP-xxx
      | tc479541/test-buildconfig-xxx.json |
```

Reply:
How can we skip scenario outlines?
Changing comment location might be tricky and is uglier IMO. Both are acceptable though.

Or might be easier to just develop out own formatter :) I dunno.

Reply:
having the comment line in the same place is clearer to me.